jsx/Card.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * This file contains React component for Card
+ *
+ * @author Zaliqa Rosli
+ * @version 1.0.0
+ *
+ */
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+
+import Panel from 'jsx/Panel';
+
+/**
+ * Card component - Wraps children in a customizable card
+ *
+ */
+class Card extends Component {
+  constructor(props) {
+    super(props);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(e) {
+    this.props.onClick(e);
+  }
+
+  render() {
+    const cursorStyle = this.props.onClick ? {
+      cursor: 'pointer',
+    } : null;
+    const divStyling = {
+      marginLeft: '5px',
+      marginRight: '5px',
+    };
+    return (
+      <div style={divStyling}>
+        <Panel
+          id={this.props.id}
+          title={this.props.title}
+          initCollapsed={this.props.initCollapsed}
+        >
+          <div
+            onClick={this.handleClick}
+            style={cursorStyle}
+          >
+            {this.props.children}
+          </div>
+        </Panel>
+      </div>
+    );
+  }
+}
+
+Card.propTypes = {
+  id: PropTypes.string.isRequired,
```

Comment:
I'm trying to use this component and I don't see why `id` is required. It doesn't seem to be declared are required by the Panel that it's being passed to, so why add this requirement for cards?

Reply:
Id is required for collapsing the intended panel. If all the cards on one page have panel component with id 'default-panel', trying to collapse any one card will always collapse the first card

Reply:
I don't understand why id is required for collapsing the panel, the collapsed state is in the state of the panel. The fact that it's required and if it's not passed or distinct the wrong panel is collapsed is why I find it confusing that you need to pass an id when trying to use this card.

Reply:
The code for collapsing the panel is in the panel component and it targets an element with the given id. If there are multiple elements with the same Id i.e. the default one, then the collapsing function can't select the correct element

Reply:
Okay, I guess that's a separate issue that should be fixed outside of this PR.